README.md

```diff
@@ -1,43 +1,58 @@
+- Proof request (what you want to prove, including proof configuration, and optionally membership lists, external nullifier and watermark): [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-verifier/app/hooks/useProofRequest.ts)
+- How to create GPC proofs (based on the proof request): [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-prover-client/app/util/generateProof.ts)
+- How to verify GPC proofs: [code](https://github.com/proofcarryingdata/zukyc/blob/main/apps/gpc-verifier/app/util/verifyProof.ts)
 
 ## What's inside?
 
-This Turborepo includes the following packages/apps:
+This Turborepo includes the following apps:
 
-### Apps and Packages
+### Apps
 
-- `docs`: a [Next.js](https://nextjs.org/) app
-- `web`: another [Next.js](https://nextjs.org/) app
-- `@repo/ui`: a stub React component library shared by both `web` and `docs` applications
-- `@repo/eslint-config`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
-- `@repo/typescript-config`: `tsconfig.json`s used throughout the monorepo
+- `pod-issuer-client`: http://localhost:3000, deployed at https://zukyc-issuer-client.vercel.app. It includes:
+  - [ZooGov](https://zukyc-issuer-client.vercel.app/gov): Govenment ID POD issuer.
+  - [ZooDeel](https://zukyc-issuer-client.vercel.app/deel): Paystub POD issuer.
+  - [ZooAdmin](https://zukyc-issuer-client.vercel.app/debug): For debugging purposes, it allows you to generate ID PODs and Paystub PODs with the information you provide.
+- `pod-issuer-server`: http://localhost:8080, deployed at https://zukyc-issuer-server.vercel.app.
+  - This is the Express server for pod-issuer-client. It provides the APIs to issues PODs to the user.
+- `gpc-prover-client` (ZooKyc): http://localhost:3002, deployed at https://zukyc-gpc-prover-client.vercel.app.
+  - This is a Next.js app which uses the GPC library to generate ZK proofs. GPCs can prove properties of one POD or serveral PODs together. PODs can be proven to be owned by the prover, using their Semaphore identity.
+- `gpc-verifier` (ZooLender): http://localhost:3001, deployed at https://zukyc-gpc-verifier.vercel.app.
+  - This is a Next.js app which uses the GPC library to verify ZK proofs.
+  - I didn't build a server side app for this, because the verification code would be similar. But in reality, the proof needs to be verified on the server side too.
 
-Each package/app is 100% [TypeScript](https://www.typescriptlang.org/).
+### Turborepo
 
-### Utilities
+This project was bootstrapped with [Turborepo starter](https://github.com/vercel/turbo/tree/main/examples/basic) by running the following command:
 
-This Turborepo has some additional tools already setup for you:
+```sh
+npx create-turbo@latest -e with-yarn
+```
 
-- [TypeScript](https://www.typescriptlang.org/) for static type checking
-- [ESLint](https://eslint.org/) for code linting
-- [Prettier](https://prettier.io) for code formatting
+## Getting started
```

Comment:
This is missing an initial step to run `yarn` to fetch packages.  Without that the `turbo` command is not found when I try to build.

Reply:
Another missing step seems to be creating a .env file:
```
/Users/artwyman/dev/zukyc/apps/pod-issuer-server/dist/index.js:9
pod-issuer-server:dev:     throw new Error("expect process.env.GOV_EDDSA_PRIVATE_KEY");
pod-issuer-server:dev:     ^
pod-issuer-server:dev: 
pod-issuer-server:dev: Error: expect process.env.GOV_EDDSA_PRIVATE_KEY
pod-issuer-server:dev:     at Object.<anonymous> (/Users/artwyman/dev/zukyc/apps/pod-issuer-server/dist/index.js:9:11)
pod-issuer-server:dev:     at Module._compile (node:internal/modules/cjs/loader:1358:14)
pod-issuer-server:dev:     at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
pod-issuer-server:dev:     at Module.load (node:internal/modules/cjs/loader:1208:32)
pod-issuer-server:dev:     at Module._load (node:internal/modules/cjs/loader:1024:12)
pod-issuer-server:dev:     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
pod-issuer-server:dev:     at node:internal/main/run_main_module:28:49
pod-issuer-server:dev: 
pod-issuer-server:dev: Node.js v20.15.0
```

Reply:
I found the .env.example.  After setting the two new private keys, I still apparently need some more:
```
pod-issuer-server:dev: Error: [Upstash Redis] The 'url' property is missing or undefined in your Redis config.
pod-issuer-server:dev:     at new o (/Users/artwyman/dev/zukyc/node_modules/@upstash/redis/nodejs.js:1:826)
pod-issuer-server:dev:     at new VercelKV (/Users/artwyman/dev/zukyc/node_modules/@vercel/kv/dist/index.cjs:5:16)
pod-issuer-server:dev:     at createClient (/Users/artwyman/dev/zukyc/node_modules/@vercel/kv/dist/index.cjs:61:10)
pod-issuer-server:dev:     at Object.<anonymous> (/Users/artwyman/dev/zukyc/apps/pod-issuer-server/dist/stores/deel.js:10:43)
pod-issuer-server:dev:     at Module._compile (node:internal/modules/cjs/loader:1358:14)
pod-issuer-server:dev:     at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
pod-issuer-server:dev:     at Module.load (node:internal/modules/cjs/loader:1208:32)
pod-issuer-server:dev:     at Module._load (node:internal/modules/cjs/loader:1024:12)
pod-issuer-server:dev:     at Module.require (node:internal/modules/cjs/loader:1233:19)
pod-issuer-server:dev:     at require (node:internal/modules/helpers:179:18)
```

Reply:
If there's a hard-coded .env we can safely include in the repo, that would be best.  Otherwise this readme should give instructions on how to create one.  Perhaps in the form of a script which can auto-generate one, so that users can simply run `yarn generate-test-env`?